node/src/lib.rs

```diff
@@ -151,6 +151,7 @@ impl Node {
             /* tx_consensus */ tx_new_certificates,
             /* rx_consensus */ rx_feedback,
             /* dag */ dag,
+            tx_feedback,
```

Comment:
> Shouldn't we modify primary.rs to receive the output pipe from block_remover and feed it into the GarbageCollector module?
@asonnino basically the channel we are passing to the `block_remover` is the `tx_consensus`, which is consumed by the [GarbageCollector](https://github.com/MystenLabs/narwhal/blob/3257759165a2f9ec09157509294146adcc0c2393/primary/src/primary.rs#L199) module - so that's already there and ready to consume the deleted certificates.

It seems that we are also using the channel name `tx_feedback` and `tx_consensus` interchangeably, so I'll refactor this to stick only to one name - `tx_consensus`